fe1-web/src/features/wallet/objects/__tests__/RollCallToken.test.ts

```diff
@@ -0,0 +1,23 @@
+import { mockKeyPair, mockLao, mockRC } from '__tests__/utils';
+import { PopToken } from 'core/objects';
+
+import { RollCallToken } from '../RollCallToken';
+
+const mockToken = PopToken.fromState(mockKeyPair.toState());
+
+describe('Roll call token object', () => {
+  it('throws when token is undefined', () => {
+    expect(() => new RollCallToken({ token: undefined })).toThrow();
```

Comment:
```suggestion
    expect(() => new RollCallToken({ token: undefined })).toThrow(Error);
```
The same goes for the other ones. It allows us to check if the correct type of error is thrown.